x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension.tsx

```diff
@@ -5,45 +5,535 @@
+                            malware: checkboxMalwareChecked,
+                          },
+                        },
+                        eventFilters: {
+                          nonInteractiveSession: radioInteractiveSelected,
+                        },
+                      },
+                    },
+                  },
+                },
+              ],
+            },
+          });
+        } else {
+          onChange({
+            isValid: true,
+            updatedPolicy: {
+              ...newPolicy,
+              inputs: [
+                {
+                  enabled: true,
+                  streams: [],
+                  type: ENDPOINT_INTEGRATION_CONFIG_KEY,
+                  config: {
+                    _config: {
+                      value: {
+                        type: e?.target?.value,
+                        endpointConfig: {
+                          preset: radioEndpointOption,
+                        },
+                      },
+                    },
+                  },
+                },
+              ],
+            },
+          });
+        }
+      },
+      [
+        onChange,
+        radioEndpointOption,
+        checkboxMaliciousChecked,
+        checkboxMalwareChecked,
+        newPolicy,
+        radioInteractiveSelected,
+      ]
+    );
+
+    const onChangeRadio = useCallback(() => {
+      setRadioInteractiveSelected(!radioInteractiveSelected);
+      setRadioComprehensiveSelected(!radioComprehensiveSelected);
+      setInteractive(radioComprehensiveSelected);
+    }, [radioInteractiveSelected, radioComprehensiveSelected, setInteractive]);
+
+    const onChangeRadioEndpoint = useCallback(
+      (e) => {
+        setRadioEndpointOption(e.target.value);
+        setPreset(e.target.value);
+      },
+      [setPreset]
+    );
+    ///
+
+    const onChangeMalwareCheckbox = useCallback(() => {
+      setCheckboxMalwareChecked(!checkboxMalwareChecked);
+      setPreventions(checkboxMaliciousChecked, !checkboxMalwareChecked);
+    }, [checkboxMalwareChecked, checkboxMaliciousChecked, setPreventions]);
+
+    const onChangeRansomwareCheckbox = useCallback(() => {
+      setCheckboxMaliciousChecked(!checkboxMaliciousChecked);
+      setPreventions(!checkboxMaliciousChecked, checkboxMalwareChecked);
+    }, [checkboxMalwareChecked, checkboxMaliciousChecked, setPreventions]);
+
     return (
-      <>
-        <EuiSpacer size="m" />
-        <EuiCallOut data-test-subj="endpointPackagePolicy_create" iconType="iInCircle">
-          <EuiText size="s">
-            <p>
-              <FormattedMessage
-                id="xpack.securitySolution.endpoint.ingestManager.createPackagePolicy.endpointConfiguration"
-                defaultMessage="We'll save your integration with our recommended defaults. You can change this later by editing the Elastic Defend integration within your agent policy."
+      <EuiForm component="form">
+        <SelectConfigurationSettingsText>
```

Comment:
You probably should be using  [EuiFormRow](https://elastic.github.io/eui/#/forms/form-layouts#form-and-form-rows) to layout your form so that spaceing/margins, etc are used